.github/workflows/CI.yml

```
@@ -0,0 +1,36 @@
+name: MODDY DEV CI
+
+on:
+  push:
+    branches: [ "develop" ]
+  pull_request: 
+    branches: [ "develop" ]
+
+jobs:
+  build: 
+    runs-on: ubuntu-22.04 
+    env:
+      working-directory: moddy-server
+
+    steps:
+    - name: 체크아웃
+      uses: actions/checkout@v3
+
+    - name: Set up JDK 17 
+      uses: actions/setup-java@v3
+      with:
+        distribution: 'corretto'
+        java-version: '17'
+
+    - name: application.yml 생성
+      run: | 
+          cd src/main/resources 
+          echo "${{ secrets.APPLICATION }}" > ./application.yml
+      working-directory: ${{ env.working-directory }}
```

p4
```suggestion
    - name: application.yml 생성
      run: | 
          cd /src/main/resources 
          echo "${{ secrets.APPLICATION }}" > ./application.yml
      working-directory: ${{ env.working-directory }}
```
호옥시 에러가 난 이유가 `src/main/resources` 에서 src 앞에 `/`가 빠졌기 때문일까.. 추측해봅니다..